editoast/osm_to_railjson/README.md

```diff
@@ -0,0 +1,17 @@
+# Converter from OSM to raijson
+
+## Usage
+
+Example for Germany:
+
+1. Download OSM's Germany data (~4Gb `germany-latest.osm.pbf`) from
+    https://download.geofabrik.de/europe/germany.html
+2. Launch osrd conversion (release build of editoast and conversion can be long):
```

Comment:
```suggestion
2. Launch conversion (release build of editoast and conversion can be long):
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/8825/commits/4722e8406c7d1ba4955e09509372e73c27fad703